plugin/CactbotEventSource/CactbotEventSource.cs

```diff
@@ -268,14 +268,31 @@ public override void SaveConfig(IPluginConfig config)
       // Log this for now as there will likely be a lot of questions, re: user directories.
       if (Config.UserConfigFile != null)
         LogInfo(Strings.CactbotUserDirectory, Config.UserConfigFile);
-
+      var gameRegion = Machina.FFXIV.Headers.Opcodes.OpcodeManager.Instance.GameRegion;
```

Comment:
Is there a reason to check the language below? Should this always use only the game region instead of game region + language?

Reply:
This is to load different opcode

Reply:
it can also ues in ngld https://github.com/ngld/OverlayPlugin/blob/1a4232e189204c15f07a57cdeb48afa8b6c37990/OverlayPlugin.Core/EventSources/EnmityEventSource.cs#L73

Reply:
Sorry, I was trying to suggest something more like:

```cs
if (gameRegion==Machina.FFXIV.GameRegion.Chinese) {
  ffxiv_ = new FFXIVProcessCn(this);
} else if (gameRegion==Machina.FFXIV.GameRegion.Korean) {
  ffxiv_ = new FFXIVProcessKo(this);
} else {
  ffxiv_ = new FFXIVProcessIntl(this);
}
```

...without checking the language setting in the ffxiv act plugin.  If we have the game region, I don't think the language matters anymore.  Does that make sense?